### ISSUE TITLE
is_valid function return false if payload has the records more than one.

### ISSUE THREAD

**Issue by koji-k** [sporto/jwto#2]
Following code works correct.(`is_valid` returns true)

```ocaml
let payload=[("user","koji")];;
let Ok encoded = Jwto.encode Jwto.HS256 "secret" payload;;
let Ok decoded = Jwto.decode encoded;;
assert (Jwto.is_valid "secret" decoded = true);;
```

But, If payload have more than one tuples, `is_valid` return false every time.
Will be returned false from is_valid when following code executes.

```ocaml
let payload=[("user","koji"); ("age", "34")];;
let Ok encoded = Jwto.encode Jwto.HS256 "secret" payload;;
let Ok decoded = Jwto.decode encoded;;
assert (Jwto.is_valid "secret" decoded = false);;

let payload=[("user","koji"); ("age", "34"), ("language", "japanese")];;
let Ok encoded = Jwto.encode Jwto.HS256 "secret" payload;;
let Ok decoded = Jwto.decode encoded;;
assert (Jwto.is_valid "secret" decoded = false);;
```

Is this expected behavior?

**Comment by koji-k** [comment]
I found maybe the cause.
I would like to create PR.

**Comment by sporto** [comment]
https://github.com/sporto/jwto/pull/3

**Comment by sporto** [comment]
Published 0.2.1 https://github.com/ocaml/opam-repository/pull/15771

**Comment by koji-k** [comment]
thank you!
